01-node-tutorial/answers/writeWithPromisesAwait.js

```diff
@@ -0,0 +1,32 @@
+const { writeFile, readFile } = require("fs").promises
+
+// Async function writer
+const writer = async () => {
+    try {
+        // Writing 3 lines to temp.txt
```

Comment:
Requires changes:
I believe the assignment wanted you to write to the file 3 times, instead of using `\n`. This would resemble the `writeWithPromisesThen` example, where you write to the same file 3 times as well.
```js
await writeFile("...", "...");
await writeFile("...", "...", { flag: "a" });
await writeFile("...", "...", { flag: "a" });
```